CMakeLists.txt

```diff
@@ -52,7 +52,9 @@ endif()
 
 add_library(ut INTERFACE)
 
-set(INCLUDE_INSTALL_DIR include/${PROJECT_NAME}-${PROJECT_VERSION}/include)
+if(NOT DEFINED INCLUDE_INSTALL_DIR)
+  set(INCLUDE_INSTALL_DIR include/${PROJECT_NAME}-${PROJECT_VERSION}/include)
+endif()
```

Comment:
Why not just use cache variables and save the `if/endif`?
(Not to mention `GNUInstallDirs` which provides a de-factor standard variable name, `CMAKE_INSTALL_INCLUDEDIR`.)